assets/js/helpers/boot.js

```diff
@@ -76,3 +78,16 @@ $(document).ready(function() {
   });
   window.bootTopic();
 });
+
+getScript = function(src, callback) {
```

Comment:
Ok, I'm closing this PR then and will take a look because @louisdelanoeye will not work either because of the reason mentioned above,